x-pack/plugins/actions/server/actions_client.test.ts

```diff
@@ -1214,6 +1257,11 @@ describe('getBulk()', () => {
           },
         },
       ],
+      connectorTokenClient: new ConnectorTokenClient({
+        unsecuredSavedObjectsClient: savedObjectsClientMock.create(),
+        encryptedSavedObjectsClient: encryptedSavedObjectsMock.createClient(),
+        logger,
+      }),
     });
     const result = await actionsClient.getBulk(['1', 'testPreconfigured']);
     expect(result).toEqual([
```

Comment:
Can't comment on the exact line, but should add an `expect` to the `delete` test to test that `deleteConnectorTokens` is called